app/frontend/js/apps/SchemaApp/SchemaApp.jsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { useSelector } from "react-redux";
+
+import { map } from 'lodash';
+
+import SchemaFieldNavigationPanel from "./components/SchemaFieldNavigationPanel";
+import SchemaField from './components/SchemaField';
+
+// actions from state
+
+import { selectSchemaFieldIds } from '~/js/features/SchemaApp/SchemaFieldsSlice';
+
+const SchemaApp = () => {
+  const schemaFieldIds = useSelector(selectSchemaFieldIds);
+
+  return (
+    <>
```

Comment:
is this empty tag a redux thing or an accident? 🤔

Reply:
This is a way of returning multiple HTML elements in one component without having to wrap them in an empty div :)